optimum/intel/openvino/trainer.py

```diff
@@ -136,6 +136,25 @@
 NNCF_LOG_FILE_NAME = "nncf_output.log"
 
 
+DEFAULT_QUANTIZATION_CONFIG = {
+    "algorithm": "quantization",
```

Comment:
@echarlaix, the idea is to keep the config inside the example for a while. Once we are ready with a "smooth" transition from PTQ to QAT in NNCF we will revise it as well.

Reply:
ok you mean keeping the examples https://github.com/huggingface/optimum-intel/blob/v1.16.0/examples/openvino/text-classification/run_glue.py#L539 compatible with the `compression` argument for some time?

Reply:
also let me know when the PR is ready for review @nikita-savelyevv